components/Footer/LinkedinIconLink.tsx

```diff
@@ -1,6 +1,11 @@
+"use client" 
+
 import React from 'react';
+import { useStrokeColor } from './useStrokeColor'
```

Comment:
This is an innovative approach but then we have to lose the SSR benfits.

see this altrernative

![image](https://github.com/Maakaf/maakaf-website/assets/12433441/54b904e2-7053-4af4-abde-24f013bca409)

Reply:
Changes applied